web/middleware/authMiddleware.ts

```diff
@@ -19,7 +19,9 @@ export class authMiddleware implements MiddlewareInterface {
          
             try{
                 var decodedToken = jwt.decode(authorizationString[1], secret);
-                request.params.authUserId = decodedToken.authUserId;    
+                request.params = {
```

Comment:
Hi @mrtomic this code piece will override any request.params coming from client and set just userId, am I correct or am I missing something?